types/src/cl_type.rs

```diff
@@ -79,25 +82,32 @@ pub enum CLType {
     /// [`PublicKey`](crate::PublicKey) system type.
     PublicKey,
     /// `Option` of a `CLType`.
+    #[cfg_attr(feature = "datasize", data_size(skip))]
```

Comment:
This is probably minor and/or wrong, I'd consider a manual `impl DataSize for CLType` here for clarity, since we're always skipping anyway.

Reply:
As discussed offline, we can address this later if required.